src/scripts.js

```diff
@@ -212,9 +218,9 @@ function getHydrationData(placement, index) {
 function getStepData(placement, index) {
   placement.innerText = `${
     currentUser.userActivity.getWeekOfData(chosenDate, 'numSteps')[index]
-  } steps, ${
+  } steps, \n ${
```

Comment:
Is the `\n` necessary? I believe template strings preserve lines. Same for below.

Reply:
Oh hey you're right! Learn something new every day!